tidysic/settings/formatted_string.py

```diff
@@ -5,57 +5,52 @@
 
 
 class _Unit(ABC):
-
     @abstractmethod
     def write(self, taggable: Taggable) -> str:
         pass
 
 
 class _SubstitutableUnit(_Unit):
-
     def __init__(self, raw_string: str):
         pattern = r"(\*?)(.*)\{(\w*)(\:.+)?\}(.*)"
         match = re.fullmatch(pattern, raw_string)
 
         if match is None:
             raise ValueError(f"Could not parse formatted string: '{raw_string}'.")
 
-        self.is_required = (match.group(1) == "*")
+        self.is_required = match.group(1) == "*"
         self.text_before = match.group(2)
         self.tag_name = match.group(3)
         self.format_spec = match.group(4)
         self.text_after = match.group(5)
 
     def write(self, taggable: Taggable) -> str:
         value = self.get_value(taggable)
-        if value is None:
+        if value == "":
```

Comment:
or `if not value:` --> then it could be anything (e.g., `""`, `None`, `[]`, ...)